### PR TITLE
Add land-preview route

### PR DESCRIPTION
**Pull request by vyrnix9582046173**

### CODE REVIEW

File: routes/land-preview.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        y,
+      };
+    }
+  })();
+  const {query = {}} = u;
+  const cache = !query['nocache'];
+  if (spec) {
+    const {z, x, y} = spec;
+    console.log('preview request', {z, x, y});
+    const key = `${z}/${x}/${y}`;
```
Comment (avaer) [inline routes/land-preview.js]: Maybe prefix with `parcels/`? So it is easy to find in S3.

File: routes/land-preview.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        let timeout;
+        const t = new Promise((accept, reject) => {
+          timeout = setTimeout(() => {
+            reject(new Error('timed out'));
+          }, 10 * 1000);
+        });
+
+        await Promise.race([
+          (async () => {
+            await page.goto(`https://app.webaverse.com/map.html?x=${x}&y=${y}&sw=16&sh=${z}&dw=128&dh=128&dst=http://${PREVIEW_HOST}:${PREVIEW_PORT}/` + index);
```
Comment (avaer) [inline routes/land-preview.js]: If we are making `z` the zoom level, then it should probably be `sw=${z}` as well.

These arguments control how many meters a tile is, and you probably want it to be a square tile, meaning `sw` and `sh` are equal.

File: routes/land-preview.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+const zlib = require('zlib');
+const child_process = require('child_process');
+const mime = require('mime');
+
+const {getObject, putObject} = require('../aws.js');
+const puppeteer = require('puppeteer');
+const browserManager = require('../browser-manager.js');
+
+const PREVIEW_HOST = '127.0.0.1';
+const PREVIEW_PORT = 8999;
```
Comment (avaer) [inline routes/land-preview.js]: If this has its own unique server, it should be on a different port from `land.js` or else this won't start.

**Review by avaer — COMMENTED** [review]
A couple small changes, and I think we should test locally to make sure it works as expected.

Otherwise, this looks good. 👍

**Comment by vyrnix9582046173** [comment]
errors were due to WSL, spun up an ec2 and it seems to work there... maybe not, testing again

figured it out, needed to add port to security rules

**Pull request merged**